web/basics.textile

```diff
@@ -49,24 +49,36 @@ Scala is not just a nicer Java.  You should learn it with a fresh mind- you will
 
 h3. Get Scala
 
-Scala School's examples work with "Scala 2.9.x":http://www.scala-lang.org/download/2.9.3.html . If you use Scala 2.10.x or newer, _most_ examples work OK, but not all.
+Scala School's examples should work with Scala 2.11. This is the same version that we use for services at Brigade. Please follow the <a href="https://brigade.atlassian.net/wiki/display/EN/Data+Team+Onboarding">onboarding instructions</a> to set up Java, Scala, Maven, and Thrift.
 
 h3. Start the Interpreter
 
-Start the included <code>sbt console</code>.
+You can start a scala console that includes all project dependencies by entering <code>mvn scala:console</code> into the command line from the root project directory.
 
 <pre>
-$ sbt console
+$ mvn scala:console
+[...build output...]
+Welcome to Scala version 2.11.6 (Java HotSpot(TM) 64-Bit Server VM, Java 1.7.0_71).
+Type in expressions to have them evaluated.
+Type :help for more information.
+
+scala>
+</pre>
 
-[...]
+If you don't have Maven set up, or don't have a maven project to work with, you can start the Scala REPL with <code>scala</code>.
 
-Welcome to Scala version 2.8.0.final (Java HotSpot(TM) 64-Bit Server VM, Java 1.6.0_20).
+<pre>
+$ scala
+Welcome to Scala version 2.11.4 (Java HotSpot(TM) 64-Bit Server VM, Java 1.7.0_71).
```

Comment:
uhhh i'm not really sure. seems that like maven is pulling a different version from somewhere.